instance/tests/integration/integration_instance.py

```diff
@@ -42,6 +43,9 @@
 
 # Tests #######################################################################
 
+@override_settings(
+    DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX='integ-',
```

Comment:
@bdero 

> ... the LMS preview domain is what's being used to determine the name of nova instance name by default.

Ah, I see -- that happens in [this line](https://github.com/open-craft/opencraft/blob/master/instance/models/openedx_appserver.py#L290). I don't remember why we do this, but maybe @mtyaka does? :)